feature/preview/src/main/java/com/google/jetpackcamera/feature/preview/PreviewScreen.kt

```diff
@@ -166,9 +166,10 @@ private fun ContentScreen(
 
     // displays toast when there is a message to show
     if (previewUiState.toastMessageToShow != null) {
-        ShowTestableToast(
+        TestableToast(
             modifier = Modifier.testTag(previewUiState.toastMessageToShow.testTag),
             toastMessage = previewUiState.toastMessageToShow,
+            shouldShowToast = false,
```

Comment:
Looks like this is hidden by default? Let's unhide for now so we have some visual feedback.

Reply:
done 👍🏾